aws-env.dev

```diff
@@ -0,0 +1,8 @@
+## This file store all environment variables for the DEV environment
+PROJECT = lambda2sqs
+AWS_REGION = ap-southeast-1
+DEPLOY_S3_PREFIX = lambda2sqs
+AWS_PROFILE = uneet-dev
```

Comment:
Shouldn't that be ins-dev?

Reply:
@kai is right, AWS profile is incorrect there.
My recommendation would be to define this variable as documented in my earlier comment: https://github.com/Unee-T-INS/lambda2sqs/pull/1#issuecomment-570032991